Copy edits

adamkdean opened the pull request:
Work in progress copy edits

ljpollard committed:
Update README.md

Minor copy edits. Simplification, sentence structure.

ljpollard committed:
Update README.md

ljpollard committed:
Update a-brief-history.md

Copy edits. As per TOV doc., I've shifted a lot of passive voice into active. This should make the copy feel lighter, more personable, and more direct. You'll find a fair few contractions where relevant, some formatting changes for legibility, and general simplification throughout.

ljpollard committed:
Update conversations-on-the-edge.md

Minor edits.

ljpollard committed:
Update core-team.md

Minor edits

ljpollard committed:
Update project-roadmaps.md

Minor edits. Passive voice. Note on simplification: I've seen a fair bit of "prepositionalisation" — i.e. "the current and upcoming development focus [for the platform]". Moving forward, I'll shift this to standard possessive ("the platform's current and upcoming development focus") as a matter of course, as it noticeably improves readability.

ljpollard committed:
Update the-edge-network.md

I've moved away from "distributed computing paradigm" in favour of explaining what that is on Line 11.

Line 29: "the connectivity of devices in multifarious network settings" — I've leaned into "to connect easily" as opposed to "connectivity", as it's more active and less abstract. I'll probably think this way when I come across similar instances. I've also chosen to rework "multifarious network settings", and explain what's meant by that within the copy itself.

Line 49: "you can reduce north-south transport, effectively converting it to east-west transport between devices and end users within your own estate" — could someone clarify what's meant by this for me?

ljpollard committed:
Update README.md

Minor edits

ljpollard committed:
Update content-delivery.md

Minor edits

ljpollard committed:
Update dns.md

Minor edits

ljpollard committed:
Update edit.com.md

Note: I'm noticing lots of "present continuous" — e.g. "giving you one clean interface". Generally, I'll revise away from this as it can feel passive and intangible. 

Otherwise, minor edits.

ljpollard committed:
Update edit.com.md

ljpollard committed:
Update edit.com.md

ljpollard committed:
Update edit.com.md

ljpollard committed:
Update object-storage.md

A fair few structural simplifiers. 

I think we need an ELI5 on "reed-Solomon erasure coding"

ljpollard committed:
Update object-storage.md

ljpollard committed:
Update object-storage.md

ljpollard committed:
Update object-storage.md

I have displaced the mention of redundancy to line 31, along with an explanation of what that means. 

Question: Reed-Solomon erasure coding — does this differ from Reed-Solomon error correction?

ljpollard committed:
Update README.md

We talk about usage in terms of "Individuals can do X"; I propose shifting to "You can do X". It's more inclusive, less hypothetical, and generally feels a bit friendlier.

"Purchases made in $EDGE are automatically bridged in to Edge Network and swapped for $XE." This got individual mention within the Service Purchase section, although it simply repeated what was said above. I've removed to avoid repetition.

Moderate switching of passive to active. A fair amount of simplification

ljpollard committed:
Update dev-fund.md

Minor edits

ljpollard committed:
Update README.md

ljpollard committed:
Update founding-team.md

ljpollard committed:
Update growth-fund.md

ljpollard committed:
Update staking-contract.md

ljpollard committed:
Update xe-distribution.md

ljpollard committed:
Update README.md

ljpollard committed:
Update founding-node.md

ljpollard committed:
Update founding-node.md

Minor edits

ljpollard committed:
Update node-one.md

Minor edits

ljpollard committed:
Update an-introduction-to-staking.md

Edit: clarity (19)

ljpollard committed:
Update expected-yields.md

Minor edits

ljpollard committed:
Update network-nodes.md

Minor edits. Compare line 65 to the master branch for an example.

ljpollard committed:
Update performance-weighting.md

Minor edits

ljpollard committed:
Update setting-up-a-host.md

Minor edits

ljpollard committed:
Update explorer.md

Moderate revision to Line 13, as well as other minor edits

ljpollard committed:
Update wallets.md

Moderate Passive --> Active. Minor edits.

ljpollard committed:
Update voting-rights.md

ljpollard committed:
Update an-introduction-to-governance.md

Moderate changes of passive to active voice.

josephdenne reviewed (approved):
I'm really happy with these updates. Any visual updates needed I'll do in situ in Gitbooks' interface post merge.

ljpollard committed:
Update guiding-principles.md

ljpollard committed:
Update README.md

Typos and minor edits

ljpollard committed:
Update community.md

Minor edits

ljpollard committed:
Update cryptocurrencies.md

Perform different functions --> Do different things
Other minor edits/typos
Formatting

ljpollard committed:
Update edge-nodes.md

Minor edits; simplicity

ljpollard committed:
Update edge-products.md

Minor edits; mainly passive --> active

ljpollard committed:
Update general.md

Minor edits, typos.

ljpollard committed:
Update nodes.md

ljpollard committed:
Update products.md

Minor edits

ljpollard committed:
Update technology.md

Edits: simplicity

ljpollard committed:
Update tokens.md

Edits, passive-->active, simplicity

ljpollard committed:
Update tokens.md

ljpollard committed:
Update README.md

Minor edit

ljpollard committed:
Update depositing-usdedge.md

Minor edits

ljpollard committed:
Update withdrawing-usdxe.md

Minor edits

ljpollard committed:
Update claiming-xe.md

Minor simplifications

ljpollard committed:
Update community-guidelines.md

Minor edits; typos, casing

ljpollard committed:
Update creating-an-xe-wallet.md

Minor edits

ljpollard committed:
Update faq.md

Edits: voice, typos, simplification, casing

ljpollard committed:
Update other-issues.md

Minor edits

ljpollard committed:
Update README.md

Very minor edits, typo

ljpollard committed:
Update open-positions.md

Minor edits

ljpollard committed:
Update community-leads.md

Minor edits

ljpollard committed:
Update referral-programme.md

Minor edits
